.circleci/config.yml

```diff
@@ -50,7 +61,18 @@ jobs:
       - run: mkdir -p ./bin
       - run: wget -O ./bin/manifest-tool https://github.com/estesp/manifest-tool/releases/download/v1.0.0/manifest-tool-linux-amd64 && chmod +x ./bin/manifest-tool
       - run: echo "export GO111MODULE=on" >>$BASH_ENV
-      - run: PATH=./bin:$PATH inv push-multiarch --binaries=speaker --tag=${CIRCLE_BRANCH:-${CIRCLE_TAG}} --docker-user=metallb
+      - run: echo ${CIRCLE_PULL_REQUEST}
```

Comment:
just curious. Would it make sense to instead of `pr-$CIRCLE_PULL_REQUEST` use something like: `pr-$CIRCLE_PULL_REQUEST-<git hash>`?

I mean, if a PR is updated the pods might need to pull the image again. The manifest default [is to always pull][link] but I wouldn't be surprised if someone changed that.

Or push `pr-$CIRCLE_PULL_REQUEST-<git hash>` *and* `pr-$CIRCLE_PULL_REQUEST-latest`?

[link]: https://github.com/metallb/metallb/blob/b60c6a110d81d7f9cc4b6a2eeb36a6b583a2aaa5/manifests/metallb.yaml#L317

Reply:
My thought was that testing the current version is what is most interesting.  Good idea on updating the docs about the image pull policy.

Reply:
I updated the docs

Reply:
Thanks!